tests/uc_oss/test_uc_oss_integration.py

```diff
@@ -0,0 +1,162 @@
+import os
+import subprocess
+import sys
+
+import pandas as pd
+import pytest
+from sklearn import datasets
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+import mlflow
+
+from tests.helper_functions import get_safe_port
+from tests.tracking.integration_test_utils import _await_server_up_or_die
+
+
+@pytest.mark.skipif(
+    "UC_OSS_INTEGRATION" not in os.environ,
+    reason="This test is only valid w/in the github workflow integration job",
+)
```

Comment:
```suggestion
pytestmark = pytest.mark.skipif(
    "UC_OSS_INTEGRATION" not in os.environ,
    reason="This test is only valid w/in the github workflow integration job",
)
```

Reply:
How will this actually trigger on the test like this?  It looks like in other tests, we do an `@var_name` annotation to indicate that it needs to be executed.

Reply:
Looks like this does work.   Seems very magical. 🙂